bundles/framework/publisher2/view/PublisherSidebar.js

```diff
@@ -66,49 +65,8 @@ class PublisherSidebar {
 
         ReactDOM.render(content, container[0]);
 
-        const accordion = Oskari.clazz.create('Oskari.userinterface.component.Accordion');
-
-        // Separate handling for RPC and layers group from other tools
-        // layers panel is added before other tools
-        // clear rpc/layers groups from others for looping/group so they are not listed twice
-        delete this.publisherTools.groups.rpc;
-        delete this.publisherTools.groups.layers;
-
-        // separate tools that support react from ones that don't
-        const reactGroups = ['tools', 'data', 'statsgrid'];
-        const reactGroupsTools = {};
-        // create panel for each tool group
-        Object.keys(this.publisherTools.groups).forEach(group => {
-            // tools is neither a jquery panel or a react-group-panel.
-            if (group === 'tools') {
-                return;
-            }
-
-            const tools = this.publisherTools.groups[group];
-            if (reactGroups.includes(group)) {
-                // panels with react groups handled after this
-                reactGroupsTools[group] = tools;
-            }
-        });
-
-        Object.keys(reactGroupsTools).forEach(group => {
-            const tools = reactGroupsTools[group];
-            // TODO: make data/statsgrid similar to the way maptools is rendered and get rid of this here implementation.
-            const toolPanel = Oskari.clazz.create('Oskari.mapframework.bundle.publisher2.view.PanelReactTools', tools, group);
-            const hasToolsToShow = toolPanel.init(this.data);
-            this.panels.push(toolPanel);
-            if (hasToolsToShow) {
-                const panel = toolPanel.getPanel();
-                panel.addClass('t_tools');
-                panel.addClass('t_' + group);
-                accordion.addPanel(panel);
-            }
-        });
-
-        // -- render to UI and setup buttons --
-        const contentDiv = container.find('div#jqueryAccordions');
-        accordion.insertTo(contentDiv);
         // disable keyboard map moving whenever a text-input is focused element
+        const contentDiv = container.find('div#publisherWrapperDiv');
```

Comment:
Looks like the only reason to have a reference is to disable keyboard navigation of map when typing on input-fields. Let's try removing the lines instead and see if it's still needed or if we still need to prevent event bubbling from the React-based inputs.